js/binance.js

```diff
@@ -1280,13 +1280,17 @@ module.exports = class binance extends Exchange {
             }
             const trading = this.safeValue (entry, 'trading');
             const active = (isWithdrawEnabled && isDepositEnabled && trading);
+            const deposit = (isDepositEnabled && trading);
+            const withdraw = (isWithdrawEnabled && trading);
```

Comment:
In this case we don't need the `trading` flag in deposit and withdraw, I think. A currency can probably be depositable and withdrawable, but not tradeable. Let's change lines 1292, 1293 like this:

```
    'deposit': isDepositEnabled,
    'withdraw': isWithdrawEnabled,
```

Reply:
Yes, make sense!